rest/model/notifications.go

```diff
@@ -0,0 +1,59 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model/notification"
+	"github.com/pkg/errors"
+)
+
+type APIEventStats struct {
+	LastProcessedAt            time.Time            `json:"last_processed_at"`
+	NumUnprocessedEvents       int                  `json:"unprocessed_events"`
+	PendingNotificationsByType apiNotificationStats `json:"pending_notifications_by_type"`
+}
+
+func (n *APIEventStats) BuildFromService(h interface{}) error {
+	stats, ok := h.(*notification.NotificationStats)
+	if !ok {
+		return errors.New("can't convert unknown type to APIEventStats")
+	}
+
+	return n.PendingNotificationsByType.BuildFromService(stats)
```

Comment:
How do LastProcessedAt and NumUnprocessedEvents get set?

Reply:
They don't get set by BuildFromService. The fields are set manually, see GetNotificationStats